server/listeners/eventHandlers/prisonEvents/index.ts

```diff
@@ -21,6 +23,10 @@ export default function buildEventHandler({ licenceService, prisonerService }: S
         case 'CONFIRMED_RELEASE_DATE-CHANGED':
           datesChangedEventHandler.handle(eventMessage).catch(error => logger.error(error))
           break
+        case 'OFFENDER-UPDATED':
+        case 'OFFENDER_DETAILS-CHANGED':
+          offenderDetailsChangedEventHandler.handle(eventMessage).catch(error => logger.error(error))
```

Comment:
we do have cypress tests for the other event types, we should probably have one for this?

Reply:
Ah of course we do, will do!